src/components/ChatInput/ChatInput.js

```diff
@@ -8,14 +8,16 @@ import styles from './ChatInput.module.css';
 import { EmojiPicker } from '../EmojiPicker/index';
 import RCContext from '../../context/RCInstance';
 import { useGoogleLogin } from '../../hooks/useGoogleLogin';
-import { useToastStore, useUserStore } from '../../store';
+import { useToastStore, useUserStore, useEditMessageStore } from '../../store';
```

Comment:
I think it will be better to use the useMessageStore rather than creating a new store